docs/DevGuide/WritingDox.md

```diff
@@ -128,7 +128,30 @@ previous section. One can also use (within a Doxygen comment) the form
 \endverbatim
 
 to put the expression on its own line. We also encourage you to use the latex
-env `align` for formatting these multiple-line equations.
+env `align` for formatting these multiple-line equations. Please prefer the
+`align` environment over the `equnarray` environment. When using out-of-line
```

Comment:
Suggest linking to https://texfaq.org/FAQ-eqnarray here on why not to use eqnarray